### PR TITLE
Article/cloudnative hpc

### PR DESCRIPTION
**Pull request by MechRock-Marcelo**
requesting final review

### CODE REVIEW

File: _posts/2020-12-03-cloud-native-hpc.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+tags: cloud hpc supercompute lustre aws batch
+image: img/blog/hpc/cloud_computing_datacentre.jpg
+author: Marcelo Aravena
+---
+<center><img src="/img/blog/hpc/cloud_computing_datacentre.jpg" /></center><br/>
+
+
+
+## What is Supercompute aka High Performance Computing (HPC)? 
+HPC, which is also commonly referred to as Supercompute, is a computing platform that aggregates multiple computers into one “Supercomputer”, just like the Voltron Lions join together to make one big robot to achieve greater power and performance in order get the job done quicker, or in Voltrons case; destroy the enemy in a more efficient and effective manner.  HPC technology will help accelerate the processing of your compute intensive workloads, jobs that would normally take days or weeks to complete on a standard single computer can be completed within minutes or hours on a Supercomputer.
```
Comment (Hamo69) [inline _posts/2020-12-03-cloud-native-hpc.md:14]: Is it worth referencing the Woodside press release here around the 1 mill vCPUs?

File: _posts/2020-12-03-cloud-native-hpc.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- Predict the weather forecast
+- Quantum Physics
+- and much more
+
+Just like you need a professional race car driver to drive a formula 1 car to its maximum potential, you ideally need users of supercomputers to be experts or specialists in their field who also understand what makes HPC technology fly.
+
+With the public cloud vendors starting to offer more HPC services as a fully managed service, it has lowered the barrier of entry for many researchers, engineers and scientists to get a slice of the cloud computing capacity.  We are now finally starting to see HPC tooling in the cloud mature to help ease the process of migrating hpc workloads into the cloud.  More researchers and scientists are now realising that the public cloud is an enabler for them to help utilise vast amounts of compute power without the monumental task of building an energy efficient datacentre with hardware capable of running their workloads and having the resources to maintain it.  The costs to do it in cloud becomes more of an operational expenditure and we get the elasticity to scale up and back down to practically zero in some cases, you don't use it, you don't pay.
```
Comment (Hamo69) [inline _posts/2020-12-03-cloud-native-hpc.md:28]: > practically zero in some cases, you don't use it, you don't pay.

should be 

practically zero in some cases; you don't use it, you don't pay.

File: _posts/2020-12-03-cloud-native-hpc.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+
+### Cloud Supercomputing fighting the good fight!
+
+Supercomputing cloud platforms have come a long way in helping make available the technology to a wider audience who are looking to process large volumes of data, model complex simulations, weather forecasting or even to help accelerate the search for corona virus treatments and vaccines which will help us understand the nature of the virus and to potentially model the spread of the outbreak.  On the flip side to that, with the global use and the rapid technological advancement of Supercomputing technology, experts and world leaders are starting to get nervous about the future of global security if it’s not regulated properly.  So let’s focus on fighting the good fight! If you are interested in learning more about HPC/Supercompute in the cloud, I would love to chat with you over a few beers and coffee!  Or If you are ready to shift your HPC workloads to the cloud and need some guidance, get in touch with us at [Mechanical Rock](https://mechanicalrock.io/our-expertise)
```
Comment (Hamo69) [inline _posts/2020-12-03-cloud-native-hpc.md:116]: > a few beers and coffee!

LOL! Sounds good for me but perhaps some people may want a few beers OR a coffee

File: _posts/2020-12-03-cloud-native-hpc.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+4. AWS Batch will launch the job on EC2 SPOT Instances which have EFA(Elastic Fabric adaptor) available for high bandwidth interconnect to help keep the MPI_Wait times down. 
+5. AWS Batch will also create the compute cluster within 1 Placement group in Cluster strategy mode, meaning that all compute nodes will be within the 1 availability zone/datacentre, keeping latency between nodes low to also help keep MPI_WAIT times low.
+6. AWS Batch will also mount the /scratch filesystem(Fsx Lustre) onto the compute nodes for you. 
+
+With [Fsx for Lustre](https://aws.amazon.com/fsx/lustre/?nc=sn&loc=1) we get the option to deploy it as a "scratch" deployment type which is ideal for temporary storage and shorter-term processing of data.  The output file can be written to Fsx for Lustre and then with some AWS Magic, we are also able to read the same file from an S3 bucket that is linked to the Fsx Lustre Filesystem.
+Researcher then retrieves the output from the S3 bucket. 
+
+So there you have an example on how to run a compute intensive hpc workload in cloud, easy enough right?
```
Comment (Hamo69) [inline _posts/2020-12-03-cloud-native-hpc.md:101]: > intensive hpc workload

for consistency should probably be HPC

File: _posts/2020-12-03-cloud-native-hpc.md (hunk 3 above)
Comment (Hamo69) [inline _posts/2020-12-03-cloud-native-hpc.md:116]: > at [Mechanical Rock](https://mechanicalrock.io/our-expertise)

Perhaps direct people to the infra mod or get in touch page?

**Review by Hamo69 — APPROVED** [review]
Very minor feedback - good read!

File: _posts/2020-12-03-cloud-native-hpc.md (hunk 1 above)
Comment (MechRock-Marcelo) [reply _posts/2020-12-03-cloud-native-hpc.md]: yes, will do it now

**Comment by paulmech** [comment]
@MechRock-Marcelo  make all the `FSx` the same? some are written as `Fsx`

File: _posts/2020-12-03-cloud-native-hpc.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+<center><img src="/img/blog/hpc/hsm-storage.png" /></center><br/>
+
+
+Do you really want to maintain such a system yourself?  AWS have solved this problem by having FSx for Lustre integrate natively with Amazon S3, making it easy to access your S3 data to run data processing workloads.  FSx for Lustre transparently presents S3 objects as files and allows you to write results back to S3, which also means we now have access to a tiered storage solution with S3 lifecycle policies! **Viola!** Brilliant work AWS! Coming from a background where maintaining Lustre and HSM solutions for on-premise HPC is a 24x7 job equating to a high TCO, This is a huge win in my books...  Just a word of caution though.. Yes you now have automated your data management strategy with most of the responsibility shifting to AWS, but as we are now migrating your data to a cheaper storage tier in the cloud, please **DO** your homework and math on the costs to retrieve the data from the archive storage tiers along with potential egress charges. You have been warned **$$$**
```
Comment (paulmech) [inline _posts/2020-12-03-cloud-native-hpc.md:112]: `Viola` => `Voilà`

File: _posts/2020-12-03-cloud-native-hpc.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+Another way to help reduce the volume of data being migrated to the cloud is to have a pre-processing step at the Edge - **Hybrid Cloud or Edge Computing**, which is when Kubernetes can be a good option to consider as it can cater for a wide variety of use cases.  This will help reduce your data footprint in the cloud, where you only transfer the pre-processed(cleansed, filtered, noise reduction) data for the core computational workloads in cloud rather than “all” the raw data. You can have the raw data written to tape or disk for archiving in a secure vault somewhere.
+
+
+### Cloud Supercomputing fighting the good fight!
```
Comment (paulmech) [inline _posts/2020-12-03-cloud-native-hpc.md:117]: seems like it needs a hyphen or an `is`, e.g.

_Cloud Supercomputing - Fighting The Good Fight_

 or

_Cloud Supercomputing is Fighting The Good Fight_

**Pull request merged**